config_management/roles/docker-configuration/files/docker.conf

```diff
@@ -1,3 +1,3 @@
 [Service]
 ExecStart=
-ExecStart=/usr/bin/docker daemon -H fd:// -H unix:///var/run/alt-docker.sock -H tcp://0.0.0.0:2375 -s overlay --insecure-registry "weave-ci-registry:5000"
+ExecStart=/usr/bin/dockerd -H fd:// -H unix:///var/run/alt-docker.sock -H tcp://0.0.0.0:2375 -s overlay --insecure-registry "weave-ci-registry:5000"
```

Comment:
See https://docs.docker.com/engine/deprecated/#docker-daemon-subcommand - it was deprecated in release 1.13.0 and targeted for removal in release 17.12.  Although it stopped working in 17.06.